src/libsyntax/codemap.rs

```diff
@@ -799,7 +799,7 @@ impl CodeMap {
 
         let lo = self.lookup_char_pos_adj(sp.lo);
         let hi = self.lookup_char_pos_adj(sp.hi);
-        return (format!("{}:{}:{}: {}:{}",
+        return (format!("{}: {}:{} -> {}:{}",
```

Comment:
The space is added only to separate the span from the file name (for visibility).
The colon has been replaced with an arrow, because we already use it to separate the line number from the character number.